src/components/views/rooms/RoomSublist2.tsx

```diff
@@ -124,10 +128,25 @@ export default class RoomSublist2 extends React.Component<IProps, IState> {
         if (this.props.onAddRoom) this.props.onAddRoom();
     };
 
-    private onResize = (e: React.MouseEvent, data: ResizeCallbackData) => {
-        const direction = e.movementY < 0 ? -1 : +1;
-        const tileDiff = this.props.layout.pixelsToTiles(Math.abs(e.movementY)) * direction;
-        this.props.layout.setVisibleTilesWithin(tileDiff, this.numTiles);
+    private onResize = (
+        e: MouseEvent | TouchEvent,
+        travelDirection: Direction,
+        refToElement: HTMLDivElement,
+        delta: { width: number, height: number }, // TODO: Use NumberSize from re-resizer when it is exposed
```

Comment:
What is this? A future library feature? Maybe link to whatever it is.

Reply:
it's just a type that the package doesn't `export`.

Reply:
You could have just said the type of `onResize` was `ResizeCallback` and it'd have had access to `NumberSize`